doc/content/research_reactors/atr/butterfly_valve/butterfly_valve_model.md

```diff
@@ -0,0 +1,93 @@
+# ATR Butterfly-Valve Model using MOOSE Navier-Stokes module
+
+*Contact: Daniel Yankura, daniel.yankura@inl.gov
+
+*Model link: [ATR Butterfly-Valve Model](https://github.com/idaholab/virtual_test_bed/tree/devel/research_reactors/atr/butterfly_valve)*
+
+!tag name=ATR Butterfly-Valve Model
+     description=3D Steady state CFD model of a butterfly-vale used at ATR, using the Navier-Stokes module
+     pairs=reactor_type:LWR
+                       geometry:valve
+                       simulation_type:fluid_flow
+                       codes_used:MOOSE_NavierStokes
+                       open_source:full
+                       computing_needs:HPC
+                       fiscal_year:2024
+                       institution:INL 
+
+The advanced test reactor (ATR) at INL employs a buttefly-valve to control pressure drop across the reactor core. 
+This simulation utilized the Navier-Stokes module in MOOSE for all simulations. 
+Running this model will likely require HPC resources. The simulations setup will be described below.
+
+# Physics Models and Boundary Conditions Used
+
+All simulations used the finite volume incompressible Navier-Stokes formulas. The pipe inlet was set as a
+constant inlet velocity that corresponded to the mass-flow rate from experimental data. The outlet pressure 
+was arbitrarily set to 0. All walls (including those belonging to the valve) were set as no-slip boundaries.
+
+# Simulation Setup
+
+The valve was simulated for 5 different angles (between 0 and 42.7 degrees) where fully open is 90 degrees.
+Due to the highly turbulent nature of the simulations, the mixing-length turbulence model with an lu
+preconditioner was used. For each valve configuration and flow-rate, three meshes of different refinement levels
```

Comment:
```suggestion
Due to the highly turbulent nature of the simulations, the mixing-length turbulence model was used. For each valve configuration and flow-rate, three meshes of different refinement levels
```

Reply:
LU is a numerical implementation detail, while mixing length is a physical model